src/Sulu/Bundle/ContactBundle/Resources/public/js/components/accounts/components/content/main.js

```diff
@@ -17,7 +17,12 @@ define(function() {
                 url: '/admin/content-navigations?alias=account'
             },
             toolbar: {
-                template: 'default'
+                buttons: [
+                    'saveWithOptions',
+                    {'settings': {
+                        dropdownItems: ['delete']
+                    }}
```

Comment:
@marcelmoosbrugger we have decided now that we use following syntax:

```
buttons: {
    saveWithOptionsSpecial: {
        parent: 'saveWithOptions'
    },
    settings: {
        options: {
            dropdownItems: {
                delete: {}
            }
        }
    }
}
```

the property name (e.g. `saveWithOptionsSpecial`) of the object is the id of the button which will be delivered with the click event. additionally a callback can be defined in which the whole button object will be passed as the first parameter (with the id as property).

you can define a `parent` button and the `options` (optional) will be merged with the parent button.

if no `parent` is defined the id will be used to look for a `parent`.

everything clear @marcelmoosbrugger ?